content/docs/5.0/get-started/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,209 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.0":
+        parent: Get Started
+        weight: 1
+---
+
+## Prerequisites
+
+Ensure that you are using MacOS or Linux.
+
+{{< warning >}}
+The following steps are only for test environments.
+{{< /warning >}}
+<!-- TODO: For production environments please refer to ??? -->
+
+## Step1: Install a Playground TiKV Cluster
+
+1. Install TiUP by executing the following command:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+```
+
+2. Set the TiUP environment variables:
+
+Redeclare the global environment variables:
+
+```bash
+$ source .bash_profile
+```
+
+Confirm whether TiUP is installed:
+```bash
+$ which tiup
+```
+
+3. If TiUP is already installed, update the TiUP playground component to the latest version:
+```bash
+$ tiup update --self && tiup update playground
+```
+
+4. Use TiUP playground to start a local TiKV cluster
+```bash
+$ tiup playground --mode tikv-slim
+```
+
+5. Press `Ctrl +C` to stop the local TiKV cluster
+
+{{< info >}}
+Refer to [TiUP playground document](https://docs.pingcap.com/tidb/stable/tiup-playground) to find more TiUP playground commands.
+{{< /info >}}
+
+## Step2: Monitor the cluster
+
+Open [http://127.0.0.1:3000](http://127.0.0.1:3000) and login to the Grafana (username: admin password: admin).
+
+
+Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
+
+## Step3: Hello, World!
+
+### Python
+
+1. Install `tikv-client` python package
+
+```bash
+$ pip3 install -i https://test.pypi.org/simple/ tikv-client
```

Comment:
```suggestion
pip3 install -i https://test.pypi.org/simple/tikv-client
```

Reply:
pip3 install -i https://test.pypi.org/simple/tikv-client
ERROR: You must give at least one requirement to install (see "pip help install")

we should keep ` `.